src/BlazorWebView/samples/WebViewAppShared/StaticFilesContents.cs

```diff
@@ -6,31 +6,31 @@
 	/// </summary>
 	public static class StaticFilesContents
 	{
-		public static readonly string CustomIndexHtmlContent = @"
-<!DOCTYPE html>
-<html>
-<head>
-    <meta charset=""utf-8"" />
-    <meta name=""viewport"" content=""width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no"" />
-    <title>Blazor Desktop app</title>
-    <base href=""/"" />
-    <link href=""css/app.css"" rel=""stylesheet"" />
-</head>
+		public static readonly string CustomIndexHtmlContent = """
+			<!DOCTYPE html>
```

Comment:
For anyone else wondering, the whitespace at the start of each line here is removed. It looks at the whitespace at the start of the _last line_ and strips out all the whitespace on every line if it matches (which it does here).

More docs: https://learn.microsoft.com/dotnet/csharp/language-reference/tokens/raw-string